### PR TITLE
Enable Tab Completion on tkn

### PR DESCRIPTION
**Pull request by nkakkar81**
Procedure to enable tab completion on tkn CLI. 

JIRA - https://issues.redhat.com/browse/RHDEVDOCS-1672

This is applicable for both OCP 4.4 and 4.5. Thanks

### CODE REVIEW

File: modules/op-tkn-configuring-completion.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="op-tkn-enabling-tab-completion_{context}"]
+= Enabling tab completion
+
+After you install the `tkn` CLI, you can enable tab completion to automatically complete `tkn` commands or suggest options when you press Tab.
+
+.Prerequisites
+
+* You must have the `tkn` CLI tool installed.
```
[the remark this answers is not in the document]
Comment (VeereshAradhya) [reply modules/op-tkn-configuring-completion.adoc:12]: @piyush-garg @praveen4g0 WDYT ?

[the remark this answers is not in the document]
Comment (piyush-garg) [reply modules/op-tkn-configuring-completion.adoc:12]: I did not face this issue ever... I mostly do `source <(tkn completion bash)`

[the remark this answers is not in the document]
Comment (nkakkar81) [reply modules/op-tkn-configuring-completion.adoc]: I have added a pre-req for installing bash-completion. Request you to please review. Thanks

[the remark this answers is not in the document]
Comment (piyush-garg) [reply modules/op-tkn-configuring-completion.adoc]: lgtm

**Review by VeereshAradhya — APPROVED** [review]

File: cli_reference/tkn_cli/op-configuring-tkn.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[id="op-configuring-tkn"]
+= Configuring the OpenShift Pipelines `tkn` CLI
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: configuring-tkn
+
+toc::[]
+
+// Enabling tab completion
+include::modules/op-tkn-enabling-tab-completion.adoc[leveloffset=+1]
```
Comment (bmcelvee) [inline cli_reference/tkn_cli/op-configuring-tkn.adoc:10]: Please add an introductory statement for the assembly.

File: modules/op-tkn-enabling-tab-completion.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+.Prerequisites
+
+* You must have the `tkn` CLI tool installed.
+* You must have `bash-completion` installed on your local system.
+
+.Procedure
+
+The following procedure enables tab completion for Bash.
+
+. Save the Bash completion code to a file.
```
Comment (bmcelvee) [inline modules/op-tkn-enabling-tab-completion.adoc:19]: s/./:

File: modules/op-tkn-enabling-tab-completion.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
++
+----
+$ tkn completion bash > tkn_bash_completion
+----
+
+. Copy the file to `/etc/bash_completion.d/`.
```
Comment (bmcelvee) [inline modules/op-tkn-enabling-tab-completion.adoc:25]: s/./:

File: modules/op-tkn-enabling-tab-completion.adoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
++
+----
+$ sudo cp tkn_bash_completion /etc/bash_completion.d/
+----
++
+You can also save the file to a local directory and source it from your `.bashrc` file instead.
```
Comment (bmcelvee) [inline modules/op-tkn-enabling-tab-completion.adoc:31]: I would suggest rewriting for clarity, maybe along the lines of:
"Alternately, you can save the file to a local directory and source it from your `.bashrc` file instead."

**Comment by bmcelvee** [comment]
Just a few suggestions, and a reminder to squash your commits. :)

File: modules/op-tkn-enabling-tab-completion.adoc (hunk 5 above)
Comment (nkakkar81) [reply modules/op-tkn-enabling-tab-completion.adoc]: done

**Comment by nkakkar81** [comment]
hey @bmcelvee - I have incorporated the comments, and have squashed the commits. Request you to please merge if you are ok with the changes. Thanks :)

**Pull request merged**